### ISSUE TITLE
how to acess a scope in side a directive which is include inside the controller?

### ISSUE THREAD

**Issue by sibascodeking** [adamalbrecht/ngOnboarding#16]
<body ng-controller="myCtrl">
<onboarding-popover enabled="onboardingEnabled" steps="onboardingSteps" on-finish-callback='myCallbackFunction' step-index='onboardingIndex'></onboarding-popover>
</body>
app.controller('myCtrl',[$scope,funciton(s){
 s.test = function(){
    console.log('Hello world');
};
}]);
 app.directive('onboardingPopover', [
    'ngOnboardingDefaults', '$sce', '$timeout', '$http', '$rootScope',function(ngOnboardingDefaults, $sce, $timeout,$http,rs) {
      return {
        restrict: 'E',
        scope: {
          enabled: '=',
          steps: '=',
          onFinishCallback: '=',
          index: '=stepIndex'
        },
        replace: true,
        link: function(scope, element, attrs) {
               //how to access the test function here...
        },
        template: ""
      };
    }
  ]);

**Comment by novarac23** [comment]
Hi @sibascodeking! 

Can you maybe rephrase a question or ask it again, it's not quite clear what the issue is from the title and description given. Please provide a bit more context. Thanks! :)
